colibre/auto_plotter/cold_gas_relations.yml

```diff
@@ -350,4 +354,534 @@ H2_mass_star_formation_rate_100:
+    section: Cold Gas Fractions vs M*
+
+sfr_neutral_to_stellar_mass_100_kpc_100:
+  type: "scatter"
+  legend_loc: "upper left"
+  selection_mask: "derived_quantities.is_active_100_kpc"
+  x:
+    quantity: "derived_quantities.specific_sfr_gas_100_kpc"
+    units: "1 / Gyr"
+    start: 1e-3
+    end: 10
+  y:
+    quantity: "derived_quantities.neutral_to_stellar_mass_100_kpc"
+    units: "dimensionless"
+    start: 1e-4
+    end: 10
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 15
+    start:
+      value: 1e-3
+      units: "1 / Gyr"
+    end:
+      value: 10
+      units: "1 / Gyr"
+    lower:
+      value: 0
+      units: "dimensionless"
+    upper:
+      value: 1e99
+      units: "dimensionless"
+  metadata:
+    title: sSFR-Neutral Gas to SF Gas Fraction (100 kpc Stellar Mass)
+    caption: All galaxies are included in the median line. Fraction is HI + H_2 mass over stellar mass in a 100 kpc aperture.
+    section: Cold Gas Fractions vs sSFR
+
+sfr_hi_to_stellar_mass_100_kpc_100:
+  type: "scatter"
+  legend_loc: "upper left"
+  selection_mask: "derived_quantities.is_active_100_kpc"
+  x:
+    quantity: "derived_quantities.specific_sfr_gas_100_kpc"
+    units: "1 / Gyr"
+    start: 1e-3
+    end: 10
+  y:
+    quantity: "derived_quantities.hi_to_stellar_mass_100_kpc"
+    units: "dimensionless"
+    start: 1e-4
+    end: 10
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 15
+    start:
+      value: 1e-3
+      units: "1 / Gyr"
+    end:
+      value: 10
+      units: "1 / Gyr"
+    lower:
+      value: 0
+      units: "dimensionless"
+    upper:
+      value: 1e99
+      units: "dimensionless"
+  metadata:
+    title: sSFR-HI Gas to SF Gas Fraction (100 kpc Stellar Mass)
+    caption: All galaxies are included in the median line. Fraction is HI mass over stellar mass in a 100 kpc aperture.
+    section: Cold Gas Fractions vs sSFR
+
+sfr_h2_to_stellar_mass_100_kpc_100:
+  type: "scatter"
+  legend_loc: "upper left"
+  selection_mask: "derived_quantities.is_active_100_kpc"
+  x:
+    quantity: "derived_quantities.specific_sfr_gas_100_kpc"
+    units: "1 / Gyr"
+    start: 1e-3
+    end: 10
+  y:
+    quantity: "derived_quantities.h2_to_stellar_mass_100_kpc"
+    units: "dimensionless"
+    start: 1e-4
+    end: 1.1
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 15
+    start:
+      value: 1e-3
+      units: "1 / Gyr"
+    end:
+      value: 10
+      units: "1 / Gyr"
+    lower:
+      value: 0
+      units: "dimensionless"
+    upper:
+      value: 1e99
+      units: "dimensionless"
+  metadata:
+    title: sSFR-H2 Gas to SF Gas Fraction (100 kpc Stellar Mass)
+    caption: All galaxies are included in the median line. Fraction is H2 mass over stellar mass in a 100 kpc aperture.
+    section: Cold Gas Fractions vs sSFR
```

Comment:
Keep this consistent with the rest `Cold Gas Fractions (sSFR)`

Reply:
I don't follow in this section why we need the vertical bounding on the median line, too.